infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/transport/sample/CorfuNettyServerChannel.java

```diff
@@ -18,9 +21,9 @@ public class CorfuNettyServerChannel extends ChannelInboundHandlerAdapter {
 
     private NettyLogReplicationServerChannelAdapter adapter;
 
-    private Map<Long, ChannelHandlerContext> contextMap;
+    private Map<Pair<UuidMsg, Long>, ChannelHandlerContext> contextMap;
```

Comment:
Are your multi source/sink tests also testing the GRPC plugin?

Reply:
added GRPC plugin to the tests as well.